demo/README.md

```diff
@@ -0,0 +1,27 @@
+# Running the Deme
+
+Create a virtual environment (running virtualenv) and point to python version 3.6:
+
+```
+virtualenv --python=python3.6 ~/swingenv
```

Comment:
Assuming we're doing this in our home directory?

May want to do `./swingenv` instead, and start the guide from checking out the repo?